GravitationalFroce.py

```diff
@@ -0,0 +1,7 @@
+G = 6.67 * (10 ** -11)
+M = 6.0 * (10 ** 24)  # Mass of Earth
+m = 7.34 * (10 ** 22)  # Mass of the moon
+r = 3.84 * (10 ** 8)
+
+Fg = G*M*m/(r ** 2)
+print(Fg)
```

Comment:
Add a  text before print the variable. The value should have 2 digits.

Reply:
Done